supportedBackends/google/pipelines/v2beta/src/main/scala/cromwell/backend/google/pipelines/v2beta/PipelinesApiAsyncBackendJobExecutionActor.scala

```diff
@@ -174,6 +175,18 @@ class PipelinesApiAsyncBackendJobExecutionActor(standardParams: StandardAsyncExe
 
   import mouse.all._
 
+  private def generateDrsLocalizerManifest(inputs: List[PipelinesApiInput]): String = {
+    inputs.collect {
+      case PipelinesApiFileInput(_, drsPath: DrsPath, _, _) =>
+        s"\"${drsPath.pathAsString}\",\"${drsPath.pathAsString}\""
```

Comment:
The second arg here is supposed to be the `containerPath` of the `PipelinesApiFileInput`.

Reply:
Good catch!